src/clr-angular/layout/tabs/tab-link.directive.ts

```diff
@@ -25,14 +24,16 @@ let nbTabLinkComponents: number = 0;
 @Directive({
   selector: '[clrTabLink]',
   host: {
-    '[id]': 'tabLinkId',
+    id: 'tabLinkId',
```

Comment:
Doesn't that mean the id is now `"tabLinkId"` as a string, instead of the actual dynamic id?